### ISSUE TITLE
Wrong Link and Wrong email id?

### ISSUE THREAD

**Issue by gcdeshpande** [pyvideo/pyvideo#275]
Hi,

Please check the contributors' file. Some other website (PyTube.org) and link included.
Kindly correct the same.

Also check for email id mentioned in "Want our Google Analytics info?" Section. is the email id mentioned correct?

**Comment by logston** [comment]
No, the email ID should be updated to direct people to the Gitter channel.
